test/cli.js

```diff
@@ -117,6 +117,13 @@ describe('Duo CLI', function () {
       var src = map.sourcesContent[map.sources.indexOf('two.js')];
       assert(src.trim() == 'module.exports = \'two\';');
     });
+
+    it('should log that the map was written', function *() {
```

Comment:
this is the correct usage of the word, however *written : thing* seems incorrect